API/CLI.md

```diff
@@ -28,6 +28,14 @@ Connecting to a Tessel 2 over USB requires no special setup.
 ### LAN
 In order to authorize the device with your computer to work over a LAN connection, call `t2 provision` after connecting it via USB. This will place an SSH key on the device. Use the `t2 wifi` command as described below to connect Tessel 2 to a local network. You should now be able to access your Tessel 2 remotely.
 
+### SSH
+To connect to your Tessel 2 over SSH use the `t2 root` command.  
+*  `t2 root`
+  * `[--timeout]` Set timeout in seconds for scanning networked Tessels.
+  *  `[--key]` SSH key for authorization with your Tessel.
```

Comment:
This will default to the key path created when your Tessel is provisioned and not required for this common unless moved by the user. Not sure how to simplify that info but probably important at least to note that it's not required by default.